tests/migrator/json_imports/test_import_ill.py

```diff
@@ -14,7 +14,7 @@
 
 def test_import_ills(test_data_migration, patrons, es_clear):
     datadir = os.path.join(os.path.dirname(__file__), "data")
-    file = (open(os.path.join(datadir, "ills.json"), "r"),)
+    file = open(os.path.join(datadir, "ills.json"), "r")
```

Comment:
is this file closed at some point? Same below

Reply:
good point, I will fix it